python/cugraph-equivariant/cugraph_equivariant/nn/tensor_product_conv.py

```diff
@@ -74,26 +79,28 @@ class FullyConnectedTensorProductConv(nn.Module):
     Examples
     --------
     >>> # Case 1: MLP with the input layer having 6 channels and 2 hidden layers
-    >>> #         having 16 channels. edge_emb.size(1) must match the size of
-    >>> #         the input layer: 6
+    >>> # having 16 channels. edge_emb.size(1) must match the size of the input layer: 6
     >>>
     >>> conv1 = FullyConnectedTensorProductConv(in_irreps, sh_irreps, out_irreps,
     >>>     mlp_channels=[6, 16, 16], mlp_activation=nn.ReLU()).cuda()
     >>> out = conv1(src_features, edge_sh, edge_emb, graph)
     >>>
     >>> # Case 2: Same as case 1 but with the scalar features from edges, sources
-    >>> #         and destinations passed in separately.
+    >>> # and destinations passed in separately. This option allows a smaller GEMM
```

Comment:
@DejunL Can you check if the description here is enough to clarify the usage?

Reply:
Added a comment below. Can you also add to the input arguments `src_scalars` and `dst_scalars`'s docstring:

```
(Scalar features of source nodes.) See example usage case for details
```
so that the user knows where to look for the examples when then read the docstring